models/labels/balancer_v2/labels_balancer_v2_schema.yml

```diff
@@ -9,36 +9,36 @@ models:
       contributors: balancerlabs
     config:
       tags: ['labels', 'ethereum', 'balancer', 'pools']
-    description: "Balancer V2 liquidity pools created on Ethereum. "
+    description: 'Balancer V2 liquidity pools created on Ethereum. '
     tests:
       - dbt_utils.unique_combination_of_columns:
           combination_of_columns:
             - address
     columns:
       - &blockchain
         name: blockchain
-        description: "Blockchain"
+        description: 'Blockchain'
       - &address
         name: address
-        description: "Address of liquidity pool"
+        description: 'Address of liquidity pool'
       - &name
         name: name
-        description: "Label name of pool containg the token symbols and their respective weights (if applicable)"
+        description: 'Label name of pool containg the token symbols and their respective weights (if applicable)'
       - &category
         name: category
-        description: "Label category"
+        description: 'Label category'
       - &contributor
         name: contributor
-        description: "Wizard(s) contributing to labels"
+        description: 'Wizard(s) contributing to labels'
       - &source
         name: source
-        description: "How were labels generated (could be static or query)"
+        description: 'How were labels generated (could be static or query)'
       - &created_at
         name: created_at
-        description: "When were labels created"
+        description: 'When were labels created'
       - &updated_at
         name: updated_at
-        description: "When were labels updated for the last time"
+        description: 'When were labels updated for the last time'
 
```

Comment:
see other schemas after you merge from main for `model_name` and `label_type` column descriptors. Add to all of the below schemas too.